hooks/msghooks.py

```diff
@@ -18,6 +18,9 @@ def on_join():
         save()
         time.sleep(5)
         return "@" + data["data"]["user_name"].replace(" ", "") + " " + WELCOME
+    elif user == 347075:
```

Comment:
Magic numbers are no bueno, could you add a set `FLAG_HATER_BLACKLIST` containing Redwolf's id and then check for `user in FLAG_HATER_BLACKLIST`?